client/src/app/shared/video/video.service.ts

```diff
@@ -115,6 +117,66 @@ export class VideoService {
       .catch((res) => this.restExtractor.handleError(res))
   }
 
+  baseFeed (
+    videoPagination: ComponentPagination,
+    sort: SortField
+  ): object {
+    const pagination = this.restService.componentPaginationToRestPagination(videoPagination)
+
+    let params = new HttpParams()
+    params = this.restService.addRestGetParams(params, pagination, sort)
+
+    let feed = {}
```

Comment:
`const feed: { [ Feed in FeedFormat ] : string } = {}` or

Reply:
`const` elements have to be initialized, so this doesn't fit.

Reply:
So use `let` :)